posts/src/main/resources/templates/scripts/CollreachPosts.js

```diff
@@ -40,9 +61,16 @@ function load() {
             let response = JSON.parse(res).postsSet;
             console.log(response);
 
-            // preprocessing some data
+            if(response.length === 0){
+                console.log("No more posts......");
+                return;
+            }
+                
```

Comment:
When no more posts are left to be shown it returns without rendering further posts.